### ISSUE TITLE
When focus goes to date in calendar NVDA screen reader read it twice. It should be read as only once to avoid confusion to user

### ISSUE THREAD

**Issue by rakeshy3** [eureka2/ab-datepicker#45]
When focus goes to date in calendar NVDA screen reader read it twice. It should be read as only once to avoid confusion to user. 
![read twice in calendar](https://user-images.githubusercontent.com/11393834/57059371-aa716180-6cd2-11e9-8998-3db310acbde9.png)

**Comment by eureka2** [comment]
Hi,

I can't reproduce this issue with NVDA 2019.1.1 :
![image](https://user-images.githubusercontent.com/200478/57067337-fde1b080-6cce-11e9-9bbb-81ddca22e09e.png)

**Comment by rakeshy3** [comment]
I have tested it in chrome browser (70.0.3538.67).

**Comment by eureka2** [comment]
If you want to avoid this behavior, delete one of the two lines 717 or 718:
```
			gridCells += ' title="' + longdate + '"';
			gridCells += ' aria-label="' + longdate + '"';
```
Take a look here: https://www.deque.com/blog/text-links-practices-screen-readers/, here: https://github.com/nvaccess/nvda/issues/7841 and here: http://pauljadam.com/demos/title-aria-label.html

If you choose to remove the title attribute, there will be no more tooltip when you move the mouse.

**Comment by rakeshy3** [comment]
ok
